pipelines/cellranger/cellranger.wdl

```diff
@@ -0,0 +1,112 @@
+workflow cellranger {
+    meta {
+        description: "Analyze 3' single-cell RNA-seq data using the 10X Genomics Cellranger pipeline."
+    }
+
+    String sample_id
+    Array[File] fastqs
+    String reference_name
+    File transcriptome_tar_gz
+    Int? expect_cells
+
+    # Runtime attributes
+    String memory = "416 GB"
+    Int boot_disk_size_gb = 12
+    String disk_space = "250"
+    Int cpu = 64
+
+    parameter_meta {
+        sample_id: "Name of sample to run CellRanger count on"
+        fastqs: "Array of fastq files for running decoupled"
```

Comment:
I'm not sure I understand what this means. What's "decoupled"?

Reply:
I found this definition in another version of the cellranger wdl, but I'll take it out since it doesn't seem to mean much 😅